resources/sdk/clisdkclient/extensions/transform_data/transform_data.go

```diff
@@ -0,0 +1,53 @@
+package transform_data
+
+import (
+	"bytes"
+	"encoding/json"
+	"text/template"
+
+	"github.com/mypurecloud/platform-client-sdk-cli/build/gc/logger"
+)
+
+var (
+	TemplateFile string
+	TemplateStr  string
+)
+
+func ConvertJsonToMap(data string) map[string]interface{} {
+	var result map[string]interface{}
+	err := json.Unmarshal([]byte(data), &result)
+	if err != nil {
+		logger.Fatalf("Error unmarshalling JSON %v\n", err)
+	}
+	return result
+}
+
+func ProcessTemplateFile(mp map[string]interface{}) string {
+	path := []string{TemplateFile}
+	tmpl := myMust(template.ParseFiles(path...))
+	return Process(tmpl, mp)
+}
+
+func ProcessTemplateStr(mp map[string]interface{}) string {
+	tmpl := myMust(template.New("tmpl").Parse(TemplateStr))
+	return Process(tmpl, mp)
+}
+
+func Process(t *template.Template, vars interface{}) string {
```

Comment:
This could start with a small "p" to make it private to the package since it's not being called anywhere else